neuromation/api/parsing_utils.py

```diff
@@ -19,6 +19,29 @@ class RemoteImage:
     registry: Optional[str] = None
     cluster_name: Optional[str] = None
 
+    @classmethod
+    def new_neuro_image(
```

Comment:
In `new_external_image()` parameters are: `name`, `tag`, `registry` (the last two are optional).
In `new_neuro_image()` parameters are: `name`, `owner`, `registry`, `cluster_name`, `tag` (only `tag` is optional).
The order of arguments `tag` and `registry` is different, and there are other parameters inserted in the middle.

I propose to move `registry` after `name` and make all parameters except `name` and `registry` keyword-only. In both methods.